QUEST_LV_0100.tsv

```diff
@@ -2371,7 +2371,7 @@ QUEST_LV_0100_20150317_002370	$To the Scout (1)
 QUEST_LV_0100_20150317_002371	$I'm just here to tell the troops to assemble
 QUEST_LV_0100_20150317_002372	$Quickly run away
 QUEST_LV_0100_20150317_002373	$To the Scout (2)
-QUEST_LV_0100_20150317_002374	$I'll find the missing luggage
+QUEST_LV_0100_20150317_002374	$I'll find the missing bag
```

Comment:
Soukyuu has this covered already. I believe he made a more fitting term for it, "belongings". 
Check his PR.